frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -512,30 +512,29 @@ describe("scenarios > collection_defaults", () => {
       cy.findByText("First Collection");
     });
 
-    it("should be possible to apply bulk selection to items (metabase#14705)", () => {
+    it("should be possible to apply bulk selection to items (metabase#14705, metabase#15338)", () => {
       cy.visit("/collection/root");
+
       selectItemUsingCheckbox("Orders");
       cy.findByText("1 item selected").should("be.visible");
+
+      openEllipsisMenuFor("Orders in a dashboard");
+      cy.findByText("Pin this item").click();
+      selectItemUsingCheckbox("Orders in a dashboard", "dashboard");
+      cy.findByText("2 items selected").should("be.visible");
+
```

Comment:
@nemanjaglumac we had two repro tests for bulk selection bugs (one for the "Select all" button and one more for selecting pinned items). I suggest merging these tests into a single one. I guess there is no reason now to distinguish pinned and unpinned items when we're talking about bulk selection. So I think it would make sense to test both of them in a single test. Please share what you think!

Reply:
Did you specifically want to check that "select-all" button is including the pinned item as well? If so, I'd suggest expanding the first test and leaving the second one intact because that's a different scenario.

I don't like the fact that we're merging two separate repros, no matter how similar they may be. Let's say one of the two regresses in the future. You might have to skip the whole test and "kill" both repros just because one failed. Also, by grouping them and putting both references to the different issues in the title, some developer in the future wouldn't know which issue refers to which part of the repro without opening GitHub and searching for an issue.

15338 was meant to **only** check that you can select the pinned item by hovering above it. If you'd like to avoid code duplication, you can pin the item in `beforeEach()` hook.

Reply:
Makes sense, will fix it 👌 Thanks